main.py

```diff
@@ -8,6 +8,8 @@
 # This sets up console and file logging (should only be called once)
 logger.initialize_logging()
 
+main_log = logging.getLogger('main')
```

Comment:
A common practice for loggers in python is to do:

```python
logger = logging.getLogger(__name__)
```

This does a few things:

* This can be copy/pasted to every file and will just work.
* It makes the logger name consistent between each file
* the `__name__` part will be whatever file name it is, so in the log message it'll show what specific file the log is happening in. Making it easy to find where the logging is actually happening.

I would suggest switching to that naming convention.

Reply:
+1

Reply:
This looks like a good idea! Implemented.